service/src/test/java/tds/session/web/endpoints/SessionControllerIntegrationTests.java

```diff
@@ -31,7 +38,7 @@
 
 @RunWith(SpringRunner.class)
 @WebMvcTest(SessionController.class)
-@Import(ExceptionAdvice.class)
+@Import({ExceptionAdvice.class, tds.common.configuration.JacksonObjectMapperConfiguration.class})
```

Comment:
Do we have to fully qualify this class? Can we just import it like we do for ExceptionAdvice?